tests/unit/test_client.py

```diff
@@ -384,6 +384,59 @@ def test_oauth2_authentication_flow(attempts, sample_post_response_data):
     assert len(_get_token_requests(challenge_id)) == attempts
 
 
+@httprettified
+def test_oauth2_refresh_token_flow(sample_post_response_data):
+    token = str(uuid.uuid4())
+    challenge_id = str(uuid.uuid4())
+
+    redirect_server = f"{REDIRECT_RESOURCE}/{challenge_id}"
+    token_server = f"{TOKEN_RESOURCE}/{challenge_id}"
+
+    post_statement_callback = PostStatementCallback(redirect_server, token_server, [token], sample_post_response_data)
+
+    # bind post statement
```

Comment:
Is first post statement (binding + actual request) testing anything more than `test_oauth2_authentication_flow`? If not then I'd remove it.

Reply:
Done.